solidity/contracts/KeepRandomBeaconOperator.sol

```diff
@@ -173,7 +173,12 @@ contract KeepRandomBeaconOperator is ReentrancyGuard {
         groups.stakingContract = TokenStaking(_stakingContract);
         groups.groupActiveTime = TokenStaking(_stakingContract).undelegationPeriod();
 
-        groupSelection.ticketSubmissionTimeout = 12;
+        // ticketSubmissionTimeout is calculated as follow:
+        // - there is 11 submission rounds
+        // - submission round duration is 6 blocks
```

Comment:
I think it's to short for Ropsten.

Reply:
Can we start e.g. 10 clients against Ropsten and do some experiments around those values? We should see what are the mining lags and how many redundant tickets are submitted.

Reply:
Already done that with two clients and observed some problems when rounds had 3 blocks because ticket submission transactions were mined with bigger lag. But OK,  I will check again with 10 clients now.

Reply:
If those times are fine for mainnet, does it being too short for ropsten cause any other problems apart from redudant tickets?

Reply:
> If those times are fine for mainnet, does it being too short for ropsten cause any other problems apart from redudant tickets?

The thing is that we are not sure if those numbers are fine for mainnet. And we can't experiment on mainnet because of some other issues. We need to make sure it works well on Ropsten and add some safety margin.

Reply:
Makes sense. We could make it 4/8 for 52 blocks, or 5/5 for 60 blocks.

Reply:
@pdyraga, made some tests on Ropsten with 10 clients and round duration of 3 blocks (with 6 blocks mining lag at the end). Here are the results:

```
operator 1: 6 members / 9 tickets submitted
operator 2: 7 members / 9 tickets submitted
operator 3: 6 members / 9 tickets submitted
operator 4: 4 members / 7 tickets submitted
operator 5: 5 members / 8 tickets submitted
operator 6: 9 members / 10 tickets submitted
operator 7: 5 members / 8 tickets submitted
operator 8: 6 members / 9 tickets submitted
operator 9: 7 members / 10 tickets submitted
operator 10: 9 members / 12 tickets submitted
```

I didn't see any problems this time, so maybe we can use the round duration of 3 blocks and if we observe problems in the future we can always change it to 6 blocks. What do you think?

Reply:
Sounds reasonable, although if we get problems I'd increment the round duration from 3 -> 4 -> 5 -> 6 until problems go away, instead of jumping straight to 6; the round duration doesn't need to be the mining lag divided by an integer.

Reply:
Yep, starting with `3` makes sense especially that our result publication block step is also set to 3.

Reply:
Ok, let's do it!